echo.go

```diff
@@ -269,6 +270,7 @@ var (
 	ErrRendererNotRegistered       = errors.New("renderer not registered")
 	ErrInvalidRedirectCode         = errors.New("invalid redirect status code")
 	ErrCookieNotFound              = errors.New("cookie not found")
+	ErrInvalidCertType             = errors.New("invalid cert type, must be string or []byte")
```

Comment:
`invalid cert or key type, must be string or []byte`

Reply:
ok